open_spiel/fog/README.md

```diff
@@ -10,38 +10,14 @@ closely, but there are some important differences, which are listed below.
 The public observations correspond to the information that all the players know
 that all the players know, like upward-facing cards on a table or passing of
 time (clock ticking). The private observation is then the remaining information,
-distinct from the public observation. Thus private and public observations are
-non-overlaping. Their union corresponds to the (undistinguished) player
-observation available as `State::ObservationString()`. The union of (public and
-private informantion) and `ObservationString` should partition the game tree
-identically (even though this is not checked).
+distinct from the public observation, with the exception of time for technical
+reasons. Thus private and public observations are almost non-overlaping. 
```

Comment:
The CL is being submitted, so any change won't be submitted, but I was wondering:
It could be useful to explain what are the technical reasons to duplicate it.

Reply:
Sure, I can update this. I am working on a number of changes to the public state API so I will update the docs as well.